00-Introducing-HOOMD-blue/00-index.ipynb

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
<p>I love this organization! All the links make this convenient to navigate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>